python/cuml/tests/test_pca.py

```diff
@@ -258,12 +258,8 @@ def test_pca_inverse_transform(datatype, input_type, name, use_handle, nrows):
     assert array_equal(input_gdf, X, 5e-5, with_sign=True)
 
 
-@pytest.mark.skipif(
-    cp.cuda.driver.get_build_version() <= 11020,
-    reason="Test failing on driver 11.2",
```

Comment:
I'm curious what we saw here -- what prompted this change? Is the failure fixed?

Reply:
Answered here: https://github.com/rapidsai/cuml/pull/5318#issuecomment-1630875292